### ISSUE TITLE
Syntax highlighting broken since build 3093

### ISSUE THREAD

**Issue by sampotts** [SublimeText/Spacegray#167]
As title basically, upgraded to 3093 because the popup just wouldn't go away (needs a skip this version) and then it broke code highlighting in JS. I've tried 3094 and 3095, same deal. Looks like 3093 caused it. 

![screen shot 2015-07-31 at 3 31 10 pm](https://cloud.githubusercontent.com/assets/719092/9001356/8259894c-3799-11e5-91f7-1e9c941c4d68.png)

Suggest sticking to 3092 if anyone was thinking of upgrading...

**Comment by SamPotts** [comment]
Here's 3092... 

![Uploading Screen Shot 2015-07-31 at 3.35.01 pm.png…]()

**Comment by philipbordallo** [comment]
@SamPotts Yeah, it weirded me out as well. But from the looks of it, pre-3093 highlighting was actually incorrect and [variables are supposed to be red](https://github.com/kkga/spacegray/blob/d9da409f0d26b9dc7211aea9ff7285601dbc1efe/base16-ocean.dark.tmTheme#L125).

Might be a good idea to file an issue with [base16-builder](https://github.com/chriskempson/base16-builder/issues).

Here is a quick bandaid to get things back the way they were. Add this to *base16-ocean.dark.tmTheme*:
```xml
<dict>
	<key>name</key>
	<string>JS Variables</string>
	<key>scope</key>
	<string>source.js variable.other, variable.function.js</string>
	<key>settings</key>
	<dict>
		<key>foreground</key>
		<string>#c0c5ce</string>
	</dict>
</dict>
<dict>
	<key>name</key>
	<string>JS DOM Support</string>
	<key>scope</key>
	<string>support.type.object.dom</string>
	<key>settings</key>
	<dict>
		<key>foreground</key>
		<string>#ebcb8b</string>
	</dict>
</dict>
<dict>
	<key>name</key>
	<string>JS Properties</string>
	<key>scope</key>
	<string>constant.other.object.key.js, string.unquoted.label.js</string>
	<key>settings</key>
	<dict>
		<key>foreground</key>
		<string>#c0c5ce</string>
	</dict>
</dict>
```

**Comment by SamPotts** [comment]
Awesome. Only thing is, where do I find that file? I looked in Application Support, no dice?

**Comment by philipbordallo** [comment]
@SamPotts It's where your packages are installed and most likely in a folder named **spacegray**. On a Mac this will be _~/Library/Application Support/Sublime Text 3/Packages_.

**Comment by dlsso** [comment]
I got this message from sublime text today:


> Fixes in Spacegray 1.3.2:
> 
>   - Fixed red-heavy syntax in JS
>   - Fixed text input field alignment
>   - Fixed image link color highlighting in Markdown
>   - Fixed Oceanic Theme input field backgorund color

However, I still have red everywhere. Is the "band aid" the only thing I can do right now?
